forge-gui/res/cardsfolder/upcoming/minthara_of_the_absolute.txt

```diff
@@ -0,0 +1,12 @@
+Name:Minthara of the Absolute
+ManaCost:2 W B
+Types:Legendary Creature Elf Cleric
+PT:2/4
+K:Starting intensity:0
+T:Mode$ ChangesZone | ValidCards$ Permanent.YouCtrl | Origin$ Battlefield | Destination$ Any | TriggerZones$ Battlefield | Execute$ TrigIntensify | ActivationLimit$ 1 | TriggerDescription$ Whenever one or more permanents you control leave the battlefield, perpetually increase NICKNAME's intensity and the intensity of cards named Minthara of the Absolute in your hand, graveyard, and library by 1. This ability triggers only once each turn.
+SVar:TrigIntensify:DB$ Intensify | AllDefined$ Card.YouOwn+namedMinthara of the Absolute | DefinedDesc$ all cards they own named Minthara of the Absolute
+  S:Mode$ Continuous | Affected$ Creature.YouCtrl | AddPower$ X | SpellDescription$ Creatures you control get +X/+0, where X is CARDNAME's intensity

```

Comment:
weird whitespace at the beginning of line??

`CARDNAME's intensity` > `CARDNAME's intensity.`

SpellDescription > Description